image/rhel/konflux.Dockerfile

```diff
@@ -11,25 +10,17 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS final-base
 # TODO(ROX-20651): use content sets instead of subscription manager for access to RHEL RPMs once available. Move dnf commands to respective stages.
 FROM ubi-base AS rpm-installer
 
-ARG ROCKSDB_BUILDER_STAGE_PATH
-COPY --from=ubi-base / "$ROCKSDB_BUILDER_STAGE_PATH"
-
 ARG GO_BUILDER_STAGE_PATH
 COPY --from=go-builder-base / "$GO_BUILDER_STAGE_PATH"
 
 ARG FINAL_STAGE_PATH
 COPY --from=final-base / "$FINAL_STAGE_PATH"
 
 COPY ./scripts/konflux/subscription-manager/* /tmp/.konflux/
-RUN /tmp/.konflux/subscription-manager-bro.sh register "$ROCKSDB_BUILDER_STAGE_PATH" "$GO_BUILDER_STAGE_PATH" "$FINAL_STAGE_PATH"
+RUN /tmp/.konflux/subscription-manager-bro.sh register "$GO_BUILDER_STAGE_PATH" "$FINAL_STAGE_PATH"
 
-# gflags and snappy-devel for rocksdb-builder come from codeready-builder-for-rhel-8 repo.
```

Comment:
`gflags` and `snappy-devel` were dependencies of `rocksdb` and we installed them in both `rocksdb-builder` and `go-builder` because otherwise we could not compile Go. Now, with `rocksdb` removed, do we still need these dependencies? If not, we can make deeper cleanup removing more stuff from these lines and potentially not even using any RHEL-entitled packages for Go builder.

```
# Install packages for the Go builder stage.
RUN dnf -y --installroot="$GO_BUILDER_STAGE_PATH" install --allowerasing make automake gcc gcc-c++ coreutils binutils diffutils gflags snappy-devel zlib-devel bzip2-devel lz4-devel cmake libzstd-devel zstd jq
```

Reply:
I will tinker with this a little and see.

Reply:
Reduced a few more items.